src/Tribe/Aggregator/Record/Queue.php

```diff
@@ -206,6 +215,7 @@ public function save() {
 		$chunker->register_chunking_for( $this->record->post->ID, $key );
 
 		if ( empty( $this->items ) ) {
+			do_action( 'tribe_aggregator_queue_finished', $this );
```

Comment:
Add Docblock

Reply:
I've removed this action as we do not use; will re-add it if we really need it in the future; it's a left-over from my testing.